frontend/lib/src/ComponentMapper.tsx

```diff
@@ -0,0 +1,128 @@
+/**
+ * Copyright (c) Streamlit Inc. (2018-2022) Snowflake Inc. (2022)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+type ElementType =
```

Comment:
this is copied from the .proto file since I couldn't figure out how to import it

Reply:
```
import {Element as ElementProto } from 'path/to/proto'

type ElementType = Exclude<ElementProto["type"], undefined>
```

Reply:
nice!